### DISCUSSION TITLE
pi-Base community call Fri 2024 Oct 4 at 4pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#745] [Announcements]
Agenda:

- Cataloging "meta" properties like "hereditary" and "embeds in": https://github.com/pi-base/data/pull/777#issuecomment-2360030610
- "non-metric continua" on homepage
- how to merge one branch (e.g. `main`) into another branch in github?
- https://github.com/pi-base/data/pull/761

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by prabau** [comment]
Looking at the pi-base home page (https://topology.pi-base.org/) and Explore page (https://topology.pi-base.org/spaces), they both give a sample search of "non-metric continua".  I have a vague recollection that we had discussed that before and decided it was not an optimal example.  Can we maybe re-discuss the sample searches on these pages?

**Reply by StevenClontz** [reply]
Places to make such changes (or propose PR):

- https://github.com/pi-base/web/blob/main/packages/viewer/src/components/Home.svelte
- https://github.com/pi-base/web/blob/main/packages/viewer/src/components/Search/Examples.svelte
